lib/xcode/install.rb

```diff
@@ -687,10 +686,12 @@ def verify_app_security_assessment
     end
 
     def verify_app_cert
-      cert_info = Fastlane::Actions::VerifyBuildAction.gather_cert_info(@path.to_s)
-      apple_team_identifier_result = cert_info['team_identifier'] == TEAM_IDENTIFIER
-      apple_authority_result = cert_info['authority'].include?(AUTHORITY)
-      apple_team_identifier_result && apple_authority_result
+      begin
```

Comment:
You can remove a level of nesting here, since a `def` is an implicit `begin`:

https://forum.upcase.com/t/implicit-begin/205 

🤷‍♂

Reply:
+1, probably not a a bad idea

Reply:
I didn't know this syntax. Thanks!